docs/styling/THEME.md

```diff
@@ -408,3 +408,48 @@ Here is an example of how your files architecture could look:
 >>>> **my-component-pres.style.scss**: component style - import style.theme.scss\
 >>>> **my-component-pres.style.theme.scss**: variables - import app-styling.scss\
 >>>> ...
+
+## Technical structure (advance)
+
+The function `apply-theme` is expected a specific structure of [SCSS Map](https://sass-lang.com/documentation/values/maps/). To facilitate its generations, the function `meta-theme-to-otter` is helping to convert an [Angular Material Theme](https://material.angular.io/guide/theming) to a compatible structure.
+
+This is an example of a theme structure:
```

Comment:
I feel that the structure is not so explicit for someone who doesn't know the framework theming. You just provide an example on what a theme structure can look like but I feel that will not help them so much.

Reply:
Added JSON schema to specify the expected structure